index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/7872c08fc1.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="./dist/css/main.css">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.2/font/bootstrap-icons.css">
+  <title>Sunshine Rock</title>
+</head>
+
+<body>
+  <header>
+    <nav class="mobile-nav" id="menu-nav-ham">
+      <section class="tools-nav">
+        <ul>
+          <li><a href="#"><i class="bi bi-facebook"></i></a></li>
+          <li><a href="#"><i class="bi bi-twitter"></i></a></li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </section>
+      <section class="nav-desktop">
```

Comment:
- [ ] In the navbar the bottom bar contains a logo but the list of page links are not displayed, please consider fixing it so it can be usable on the website. (consider adding a list of page links aligned to the right)